database/dynamodb.go

```diff
@@ -29,34 +22,56 @@ type AWSCredentials struct {
 var db *dynamodb.DynamoDB
 
 func init() {
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println(err)
+		}
+	}()
 
-	env, found := os.LookupEnv(Constants["ENV"])
-	if !found {
-		// load the env values using the `setUpEnv` function.
-		utils.SetUpEnv()
-	}
+	env := os.Getenv(utils.ENV)
+	if env == utils.PROD {
+		log.Println(env, " is the env")
+	} 
 
 }
 
 func getAWSCredentials() *AWSCredentials {
 
+	var found bool
```

Comment:
These lines are trying to get the environment variables utils.REGION, utils.ACCESS_KEY, and utils.SECRET_KEY. However, the variables utils.REGION, utils.ACCESS_KEY, and utils.SECRET_KEY have not been defined yet. This means that the calls to os.LookupEnv() will return the empty string.

Reply:
https://github.com/Real-Dev-Squad/feature-flag-backend/pull/32/files#diff-ae79249906eadef6d2aa935b7a43f98abb5092136b4c421da80d2f7dc29e94cdR7 this line we have the string for `utils.REGION` i.e "REGION" and we have not defined the code that takes the env variables in production, currently its only for dev.